### ISSUE TITLE
utls with proxy

### ISSUE THREAD

**Issue by Nyx2022** [projectdiscovery/fastdialer#142]
i know it has supported utls in tls handshake,but it does not support utls with socks5 proxy support,so iadvise to add this feature

**Comment by Mzack9999** [comment]
@Nyx2022 Thanks for opening this issue. Could you provide more insight about the use case of this feature? I suppose the proxy is potentially blocking clients based on ja3?

**Comment by Nyx2022** [comment]
> @Nyx2022 Thanks for opening this issue. Could you provide more insight about the use case of this feature? I suppose the proxy is potentially blocking clients based on ja3?

yes, one situation is the proxy is potentially blocking clients based on ja3,the second one is the target server is on intranet,and we must use socks5 proxy to access it , ant the same time.the target will blocking clients based on ja3,so we also need utl to fake ja3, so it's why we need socks5+utls
